Developers/Newcomers_Guide.rst

```diff
@@ -46,7 +46,7 @@ Step 3. Picking Up an Issue
 ---------------------------
 
 Now it is time to pick an issue.
-Here is the link that will lead you to `Newcomers issues <http://tinyurl.com/coala-new>`_.
+Here is the link that will lead you to `Newcomers issues coala.io/new`_.
```

Comment:
that is not done well. this structure as in `text <link>`_ is .rst specific and it makes the ``text`` be a hyperlink to the ``link``, so you still need to add the ``<>`` before the link and after it.